scripts/index.js

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable func-names */
+let bookList = [
+  {
+    Title: 'Songs of Fire & Ice',
+    Author: 'George R. R. Martin',
+  },
+  {
+    Title: 'Lord of The Rings',
+    Author: 'J. R. R. Tolkien',
+  },
+];
+
+function storageAvailable(type) {
+  let storage;
+  try {
+    storage = window[type];
+    const x = '__storage_test__';
+    storage.setItem(x, x);
+    storage.removeItem(x);
+    return true;
+  } catch (event) {
+    return event instanceof DOMException && (event.code === 22 || event.code === 1014 || event.name === 'QuotaExceededError' || event.name === 'NS_ERROR_DOM_QUOTA_REACHED') && (storage && storage.length !== 0);
+  }
+}
+
+function updateStorage() {
+  if (storageAvailable('localStorage')) {
+    const { localStorage } = window;
+    localStorage.setItem('bookList', JSON.stringify(bookList));
+  }
+}
+
+const bookDiv = document.getElementsByClassName('book-list')[0];
+
+function populateBook() {
+  updateStorage();
+
+  bookList.forEach((book, index) => {
+    const h4 = document.createElement('h4');
+    h4.innerText = book.Title;
+
+    const p = document.createElement('p');
+    p.innerText = book.Author;
+
+    const hr = document.createElement('hr');
+
+    const removeButton = document.createElement('button');
+    removeButton.className = 'remove-button';
+    removeButton.id = index;
+    removeButton.innerText = 'Remove';
+
+    bookDiv.appendChild(h4);
+    bookDiv.appendChild(p);
+    bookDiv.appendChild(removeButton);
+    bookDiv.appendChild(hr);
+  });
+}
+
```

Comment:
- [ ] Your app does not work as expected. When I add a book and refreshed the page, the local storage is cleared and so nothing is displayed on the page. Kindly note that changes are expected to persist on the local storage and available books should be present on reload.

  A book is created | Browser is reloaded
  :---:|:---:
  ![hm-1](https://user-images.githubusercontent.com/81468675/183517518-e495d301-bfab-4b9f-85ae-ecd6f4eec23c.jpg) | ![hm-2](https://user-images.githubusercontent.com/81468675/183517513-034cc15e-1e0c-4975-b1ec-bb4df035bf3d.jpg)